vanilla-framework/vanilla-framework
# Move /spin page to a new section on /icons page

## Done
- [Spin](https://docs.vanillaframework.io/en/utilities/spin) felt odd being its own page on **Images and Media** so I moved it as a sub-section under [Icons](https://docs.vanillaframework.io/en/patterns/icons) as I felt it was more relevant there
- Moved [Spin](https://docs.vanillaframework.io/en/utilities/spin) to Icons page under a new section called **Animation**
- Deleted [Spin](https://docs.vanillaframework.io/en/utilities/spin) page under **Images and Media**

## QA
- Pull code
- Run /docs
- Check http://0.0.0.0:8104/en/patterns/icons

## Screenshot
![Screenshot 2019-04-01 at 10 31 32](https://user-images.githubusercontent.com/17748020/55317951-80c1e200-5469-11e9-8bb9-7309f230b90a.png)
